cmd/nephe-controller/main.go

```diff
@@ -125,25 +125,33 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "VirtualMachine")
 		os.Exit(1)
 	}
-	if err = (&crdv1alpha1.VirtualMachine{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "VirtualMachine")
-		os.Exit(1)
-	}
-	if err = (&crdv1alpha1.CloudEntitySelector{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "CloudEntitySelector")
-		os.Exit(1)
-	}
 
-	if err = (&crdv1alpha1.CloudProviderAccount{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "CloudProviderAccount")
-		os.Exit(1)
-	}
-
-	// Register webhook for secret
+	// Register webhook for secret.
 	mgr.GetWebhookServer().Register("/validate-v1-secret",
 		&webhook.Admission{Handler: &nephewebhook.SecretValidator{Client: mgr.GetClient(),
 			Log: logging.GetLogger("webhook").WithName("Secret")}})
 
+	// Register webhook for CloudProviderAccount Mutator.
+	mgr.GetWebhookServer().Register("/mutate-crd-cloud-antrea-io-v1alpha1-cloudprovideraccount",
+		&webhook.Admission{Handler: &nephewebhook.CPAMutator{Client: mgr.GetClient(),
+			Log: logging.GetLogger("webhook").WithName("cloudprovideraccount-resource")}})
```

Comment:
We should have uniform name (i.e either follow camel case or lower case) for logging. It can be either like L137 or like L132.
I'm okay with either of them.

Reply:
done